projects/igniteui-angular/src/lib/badge/badge.component.ts

```diff
@@ -133,6 +133,19 @@ export class IgxBadgeComponent {
     @HostBinding('class.igx-badge')
     public cssClass = 'igx-badge';
 
+    /**
+     * Sets a square shape to the badge, if `[square]` is set to `true`.
+     * By default the shape of the badge is rounded.
+     *
+     * @example
+     * ```html
+     * <igx-badge [square]="true"></igx-badge>
+     * ```
+     */
+    @HostBinding('class.igx-badge--square')
+    @Input()
+    public square = false;
```

Comment:
Is this implemented as such in the design spec? With the avatar we have `shape` in the design spec, which is covered by a `roundShape` input. Here the input is the opposite: `square`. I believe we should unify these, and maybe even cover them by the `roundness` instead of with an input? @simeonoff @sdimchevski